db/migrate/20230411032053_insert_community_to_sidebar_sections.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+class InsertCommunityToSidebarSections < ActiveRecord::Migration[7.0]
+  COMMUNITY_SECTION_LINKS = [
+    {
+      name: I18n.t("sidebar.sections.community.links.everything.content", default: "Everything"),
```

Comment:
Related to https://github.com/discourse/discourse/pull/21166/files#r1183192171 as well. The text needs to be hardcoded since translations can change over time.